migrations/sql/V2019.07.23.16.20__add_guid_to_incident_recommendations.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE mine_incident_recommendation
+ADD COLUMN mine_incident_recommendation_guid uuid    DEFAULT gen_random_uuid() NOT NULL,
```

Comment:
does this table need a guid to expose it directly publicly? seems like the integer id so it can FK to the incident seems like all this would need.

Reply:
Yeah. It gives the FE a unique identifier to pass along with an existing recommendation so that it's clear which record(s) need to be updated/soft deleted.

Reply:
Good points, sounds great.